toshi_hazard_store/scripts/ths_import.py

```diff
@@ -62,14 +62,76 @@
 hpc_manager = HazardCurveProducerConfigManager(pathlib.Path(STORAGE_FOLDER), chc_manager)
 
 
+def store_hazard(
+    hdf5_path: str,
+    config_path: str,
+    compatible_calc_id: str,
+    hazard_calc_id: str,
+    ecr_digest: str,
+    output: str,
+) -> None:
+    """
+    Extract openquake hazard curves from HDF5_PATH writing to OUTPUT in parquet format.
+
+    This function is provided for direct use (the python API) and is also
+    used by the CLI wrapper function `store_hazad_api`.
+
+    Args:
+    hdf5_path: path to the hazard realization HDF5 file.
+    config_path: path to the `oq_config.json` file.
+    compatible_calc_id: FK of the compatible calculation.
+    hazard_calc_id: FK of the hazard calculation.
+    ecr_digest: AWS ECR SHA256 digest of the hazard docker image.
+        e.g sha256:db023d95e7ec6707fe3484c7b3c1f8fd4d1c134d5a6d7ec5e939700b625293d9
+    output: path to the output file OR S3 URI.
+
+    Returns:
+        None
+    """
+    # Check paths
+    config_file_path = pathlib.Path(config_path)
+    hdf5_file_path = pathlib.Path(hdf5_path)
+    assert config_file_path.is_file(), f"config file {config_path} is not a file."
```

Comment:
use `if` and `raise` rather than `assert`. `assert` is turned off by optimization `-O` flag (https://www.reddit.com/r/Python/comments/1e33nuh/is_common_best_practice_in_python_to_use_assert/)

```suggestion
    if not config_file_path.is_file():
        raise FileNotFoundError(f"config file {config_path} is not a file.")
```